browser/extensions/extensibles/test/browser/head.js

```diff
@@ -70,6 +128,29 @@ async function openToolbarContextMenu(contextMenu, target) {
   await popupshown;
 }
 
+/**
+ * Helper to paste from clipboard
+ */
+
+async function pasteFromClipboard(browser) {
+  return await SpecialPowers.spawn(browser, [], async () => {
```

Comment:
Is async needed here since the function is returning a Promise and there's no awaits?

Reply:
We do need it as we await `SpecialPowers.spawn()`, otherwise we would have a Promise nested within another Promise.

Reply:
I see what you are saying now, I'll try removing all the async and see what happens